frontend/package.json

```diff
@@ -3,28 +3,23 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
-    "@fortawesome/fontawesome-svg-core": "^1.2.28",
-    "@fortawesome/free-regular-svg-icons": "^5.13.0",
-    "@fortawesome/free-solid-svg-icons": "^5.13.0",
-    "@fortawesome/react-fontawesome": "^0.1.9",
-    "@testing-library/jest-dom": "^4.2.4",
-    "@testing-library/react": "^9.5.0",
-    "@testing-library/user-event": "^7.2.1",
-    "babel-polyfill": "^6.26.0",
-    "cross-fetch": "^3.0.4",
-    "date-fns": "^2.13.0",
-    "node-sass": "^4.14.1",
-    "prop-types": "^15.7.2",
```

Comment:
We only had [PropTypes](https://reactjs.org/docs/typechecking-with-proptypes.html) in one component and tbh I'd rather incrementally adopt Typescript that add more of them